### ISSUE TITLE
Apple Silicon release

### ISSUE THREAD

**Issue by Fanfwe** [psychs/limechat#331]
Hi,
First of all, thanks for this very useful software.
Any chance a new build of the existing code can be published on the mac app store, with universal format for both Intel and ARM architectures ?

**Comment by tcurdt** [comment]
Here for the same thing. Any blockers? Anything to help with?

**Comment by rullinoiz** [comment]
also came to suggest a universal binary

**Comment by vk3xxx** [comment]
Yep same here.  Any ideas when a "universal" app will be available?

**Comment by Fanfwe** [comment]
I have just cloned the git repository and built it with the latest XCode. It builds fine and runs as a native Apple Silicon app.
All that is required is for the app maintainer to issue a new release and distribute it on the mac app store.

**Comment by arthur-st** [comment]
Same request here, would be wonderful to have LimeChat released as an universal app.

**Comment by aviwad** [comment]
> I have just cloned the git repository and built it with the latest XCode. It builds fine and runs as a native Apple Silicon app. All that is required is for the app maintainer to issue a new release and distribute it on the mac app store.

Can you upload your universal build as a DMG? thank you

**Comment by i0ntempest** [comment]
I've submitted this to [MacPorts](https://github.com/macports/macports-ports/commit/6e2c1e19a4ffce5d59a9cdf8147022ad176dffed). MacPorts will build the app for your architecture.

**Comment by jameshilliard** [comment]
@psychs Think you could upload a universal build? It looks like apple plans to drop support for running the intel versions on apple silicon soon so the current app store release will likely break.
